.github/test-spec.yml

```diff
@@ -22,7 +22,8 @@
   - "mpsl/lib/**/*"
 
 "CI-mesh-test":
-  - "**/*"
+  - "softdevice_controller/include/**/*"
+  - "softdevice_controller/lib/**/*"
```

Comment:
Should mpsl include and lib be added too?

Reply:
Adding them 👍